Content.Server/Medical/HealingSystem.cs

```diff
@@ -69,17 +82,40 @@ private void OnDoAfter(EntityUid uid, DamageableComponent component, HealingDoAf
         _stacks.Use(args.Used.Value, 1);
 
         if (uid != args.User)
+        {
             _adminLogger.Add(LogType.Healed,
                 $"{EntityManager.ToPrettyString(args.User):user} healed {EntityManager.ToPrettyString(uid):target} for {total:damage} damage");
+        }
         else
+        {
             _adminLogger.Add(LogType.Healed,
                 $"{EntityManager.ToPrettyString(args.User):user} healed themselves for {total:damage} damage");
+        }
 
         _audio.PlayPvs(healing.HealingEndSound, uid, AudioHelpers.WithVariation(0.125f, _random).WithVolume(-5f));
 
+        // Logic to determine the whether or not to repeat the healing action
+        args.Repeat = (HasDamage(component, healing) && !dontRepeat);
+        if (!args.Repeat && !dontRepeat)
+            _popupSystem.PopupEntity(Loc.GetString("medical-item-finished-using", ("item", args.Used)), uid);
```

Comment:
add braces to this if statement too

Reply:
Aren't one line if statements braceless?

Reply:
ok yeah, this is fine as long as the block is a single line